examples/fscloud/main.tf

```diff
@@ -87,9 +87,18 @@ module "cbr_account_level" {
     }
   }
 
-  # Demonstrates how a customized name can be set for the CBR zone
-  zone_service_ref_list = {
-    "codeengine" = "codeengine-zone-example-of-customized-zone-name"
+  # Demonstrates how a customized name and an optional location can be set for the CBR serviceRef zones
+  zone_service_ref_map = {
+    "codeengine" = {
+      zone_name           = "codeengine-zone-example-of-customized-zone-name"
+      serviceRef_location = ["au", "syd"]
```

Comment:
In this example, could we just confirm that traffic remains blocked for the other regions. Given that CBR has not an deny-by-default model backed in, I'm not 100% certain that this is not opening up the other regions.

Reply:
@vburckhardt, I did the testing with one of the scenario where I tried get the list of EN integrations.

1. Create EN instance in `us-south` region with CBR rule (enabled and scoped it with EN instance guid) with context as Schematics network zone `en-int-schematics-zone` with location ["us", "dal"].

2. So any request going from schematics workspace `geretain-en-priv` (us-south) to get the list of EN integrations, the reqest is successful.
    <img width="1281" alt="image" src="https://github.com/user-attachments/assets/4350239f-821c-435c-8309-2328be99c2e0">
     List EN integration successful
     <img width="888" alt="image" src="https://github.com/user-attachments/assets/5915ffcd-1e93-49af-a800-a70c1fa56e7b">

3. Created another Schematics workspace `man-test` (eu-gb) region, when EN list integration request comes from this `man-test` workspace, the request is getting denied.
    <img width="1291" alt="image" src="https://github.com/user-attachments/assets/b8a2cb3d-6599-4505-ba55-c9c5ae2066f1">

     List integration  failed due to CBR rule
   <img width="918" alt="image" src="https://github.com/user-attachments/assets/4e451d2f-e8f3-4baf-a9f2-93abcb1d04bf">

4. When the enforcement mode is disabled or in report mode, list integration request is successful.
---
- EN instance in `us-south` details
  <img width="1497" alt="image" src="https://github.com/user-attachments/assets/3b8516bc-863d-4c60-898e-8259ab2e58e4">

- EN CBR rule (scoped it with EN instance guid) with context as Schematics network zone `en-int-schematics-zone` with location ["us", "dal"]
   <img width="306" alt="image" src="https://github.com/user-attachments/assets/7560b139-5c83-4aaf-a4b2-6e20894e18d0">


- CBR ZONE  `en-int-schematics-zone` zone definition with locations ["us", "dal"]
  > {
  >   "name": "en-int-schematics-zone",
  >   "description": "CBR Network zone containing Schematics",
  >   "addresses": [
  >     {
  >       "type": "serviceRef",
  >       "ref": {
  >         "service_name": "schematics",
  >         "account_id": "abac0df06b644a9cabc6e44f55b3880e",
  >         "location": "us"
  >       }
  >     },
  >     {
  >       "type": "serviceRef",
  >       "ref": {
  >         "service_name": "schematics",
  >         "account_id": "abac0df06b644a9cabc6e44f55b3880e",
  >         "location": "dal"
  >       }
  >     }
  >   ],
  >   "excluded": []
  > }